apps/console-lite/src/app/components/deal-ticket/deal-ticket-balance.spec.tsx

```diff
@@ -1,13 +1,11 @@
 import React from 'react';
 import { render } from '@testing-library/react';
-import type {
-  AccountFragment,
-  DealTicketMarketFragment,
-} from '@vegaprotocol/deal-ticket';
+import type { AccountFragment } from '@vegaprotocol/deal-ticket';
 import { DealTicketBalance } from './deal-ticket-balance';
 import { Schema } from '@vegaprotocol/types';
+import type { MarketDealTicket } from '@vegaprotocol/market-list';
 
-const tDAI: DealTicketMarketFragment['tradableInstrument']['instrument']['product']['settlementAsset'] =
+const tDAI: MarketDealTicket['tradableInstrument']['instrument']['product']['settlementAsset'] =
```

Comment:
suggestion (non blocking): I think an type alias here would make this more readable, its could also be used elsewhere

 `type SettlementAsset = MarketDealTicket['tradableInstrument']['instrument']['product']['settlementAsset']`

Reply:
ok, I created some